src/components/swaptab/index.js

```diff
@@ -221,6 +221,15 @@ class SwapTab extends React.Component {
       );
     }
 
+    // TODO: Remove this once we resolve the issues
+    if (this.quoteAsset.symbol === 'BTC' && !this.quoteAsset.isLightning) {
```

Comment:
We don't need to disable lightning BTC, we just need to disable on chain BTC as both quote and base asset pairs. Our on chain BTC wallet is getting drained due to a critical bug.